stream/stream.h

```diff
@@ -119,6 +119,8 @@ CURRENT_STRUCT(StreamSchemaFormatNotFoundError) {
 template <typename ENTRY>
 using DEFAULT_PERSISTENCE_LAYER = current::persistence::Memory<ENTRY>;
 
+enum class SubscriptionMode : int { Safe = 0, Unsafe = 1 };
```

Comment:
Curious why not `bool`, if you already are in the business of specifying the values.